### PR TITLE
Azure pipeline

### PR DESCRIPTION
**Pull request by endakelly**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPO-2879

### Change description ###
Adding azure pipeline setup for repo

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  - name: env
+    displayName: Environment
+    type: string
+    default: 'PTLSBOX'
+    values:
+    - PTLSBOX
+
+variables:
+  - name: tfversion
+    value: 0.15.4
```
Comment (timja) [inline azure-pipelines.yml:33]:
```suggestion
    value: 1.0.0
```

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    value: 'sds-jenkins'
+  - name: action
+    value: ${{ parameters.Action }}
+  - name: builtFrom
+    value: $(Build.Repository.Name)
+
+stages:
+  - stage: 
+    displayName: 'Jenkins Infrastructure'
+    jobs:
```
Comment (timja) [inline azure-pipelines.yml:50]: this should use the standard pipeline from https://github.com/hmcts/cnp-azuredevops-libraries

**Review by timja — CHANGES_REQUESTED** [review]

File: azure-pipelines.yml (hunk 2 above)
Comment (endakelly) [reply azure-pipelines.yml:50]: Updated to match standard

File: azure-pipelines.yml (hunk 1 above)
Comment (endakelly) [reply azure-pipelines.yml]: Updated to match standard

**Review by timja — CHANGES_REQUESTED** [review]
missing .terraform-version file at the root symlinked into the component

**Comment by endakelly** [comment]
> missing .terraform-version file at the root symlinked into the component

Added in now

File: .terraform-version [hunk 3]
```diff
@@ -0,0 +1 @@
+1.0.0
```
Comment (timja) [inline .terraform-version:1]: this should be symlinked to the component, see the example in https://github.com/hmcts/cnp-azuredevops-libraries#required-terraform-folder-structure

**Review by timja — APPROVED** [review]

**Pull request merged**